test/sim/moves/electricterrain.js

```diff
@@ -95,4 +95,14 @@ describe('Electric Terrain', function () {
 		const resultMove = toID(battle.log[battle.lastMoveLine].split('|')[3]);
 		assert.equal(resultMove, 'thunderbolt');
 	});
+
+	it.skip(`should block Sleep before the move would have missed`, function () {
```

Comment:
This is just for status moves that cause Sleep, right? Relic Song's Sleep just gets silently blocked as normal.

Reply:
Right, only moves that Electric Terrain would "loudly" block.